### PR TITLE
Use brms-bpmsuite-platfrom-bom + remove usage of parent for individual quickstarts

### PR DESCRIPTION
**Pull request by psiroky**
@ryanzhang please take a look at these changes and let me know what you think.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -45,15 +45,12 @@
         <!-- JBoss dependency versions -->
         <version.jboss.maven.plugin>7.4.Final</version.jboss.maven.plugin>
 
-        <!-- Define the version of the JBoss BOMs we want to import to specify tested stacks. -->
+        <!-- Define the version of the JBoss EAP BOM we want to import to specify tested stacks. -->
         <version.org.jboss.bom.eap>6.4.2.GA</version.org.jboss.bom.eap>
```
Comment (ryanzhang) [inline pom.xml]: The version.org.jboss.bom.eap is for defining the jboss-javaee-6.0-with-tools.
If jboss-javaee-6.0-with-tools is really not necessary (I see it is removed), maybe this properties is also redundant?

File: stateful-ksession/pom.xml [hunk 2]
```diff
@@ -36,6 +33,14 @@
         </license>
     </licenses>
 
+    <properties>
+        <version.org.jboss.bom.brms>6.2.0-SNAPSHOT</version.org.jboss.bom.brms>
+        <version.org.jboss.bom.eap>6.4.2.GA</version.org.jboss.bom.eap>
```
Comment (ryanzhang) [inline stateful-ksession/pom.xml:38]: Should we update this one  to 6.4.3.GA since CR28 is align to 6.4.3?
I am going to do ER2 base on 6.4.3 eap dependencies.

**Comment by ryanzhang** [comment]
Thanks Petr!
The changes looks good to me. Just commented two very small issue , please double check.

File: pom.xml (hunk 1 above)
Comment (psiroky) [inline pom.xml]: Good catch. I will remove rest of the version properties as well, because they are not needed in the top-level pom.

File: stateful-ksession/pom.xml (hunk 2 above)
Comment (psiroky) [inline stateful-ksession/pom.xml:38]: We should use 6.4.3.GA, but let's do that in different PR/commit.

**Comment by psiroky** [comment]
I removed the version properties from the top-level pom as they are not needed there.

(I did that in different commit to easily see only the changes. Once merging the commits should be squashed into single one)

**Comment by psiroky** [comment]
Merged into 6.2.x
